api/appendices/provider_types.adoc

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
actually, the leading blanks in the files fix the problem asciibinder had with the [[ ]] tagged entries where they were shown in the rendered html.